src/EnergyPlus/UnitarySystem.cc

```diff
@@ -16739,6 +16747,33 @@ namespace UnitarySystems {
         return false;
     }
 
+    bool getUnitarySystemNodeNumber(EnergyPlusData &state, int const nodeNumber)
+    {
+        for (int unitarySysIndex = 0; unitarySysIndex <= state.dataUnitarySystems->numUnitarySystems - 1; ++unitarySysIndex) {
+            auto &unitarySys = state.dataUnitarySystems->unitarySys[unitarySysIndex];
+
+            int FanInletNodeIndex = state.dataFans->fans(unitarySys.m_FanIndex)->inletNodeNum;
+            int FanOutletNodeIndex = state.dataFans->fans(unitarySys.m_FanIndex)->outletNodeNum;
+
+            bool noUnitarySysOutdoorAir = false;
+            if (unitarySys.m_CoolOutAirVolFlow == 0 && unitarySys.m_HeatOutAirVolFlow == 0 && unitarySys.m_NoCoolHeatOutAirVolFlow == 0) {
+                noUnitarySysOutdoorAir = true;
+            }
+
+            if (unitarySys.m_sysType == UnitarySys::SysType::PackagedWSHP || unitarySys.m_sysType == UnitarySys::SysType::PackagedAC ||
+                unitarySys.m_sysType == UnitarySys::SysType::PackagedHP) {
+                if (noUnitarySysOutdoorAir && (nodeNumber == FanInletNodeIndex || nodeNumber == FanOutletNodeIndex ||
+                                               nodeNumber == unitarySys.AirInNode || nodeNumber == unitarySys.m_OAMixerNodes[0] ||
+                                               nodeNumber == unitarySys.m_OAMixerNodes[1] || nodeNumber == unitarySys.m_OAMixerNodes[2]) ||
+                    nodeNumber == unitarySys.m_OAMixerNodes[3] || nodeNumber == unitarySys.CoolCoilOutletNodeNum ||
+                    nodeNumber == unitarySys.HeatCoilOutletNodeNum) {
+                    return true;
+                }
+            }
+        }
+        return false;
+    }
+
```

Comment:
New function to identify nodes used by PTAC/PTHP/WSHP that don't use any OA so they can be excluded from the AFN distribution validation.